pyramid_swagger/ingest.py

```diff
@@ -62,27 +82,76 @@ def _load_resource_listing(resource_listing):
         )
 
 
-def compile_swagger_schema(schema_dir):
+def generate_resource_listing(schema_dir, listing_base):
+    if 'apis' in listing_base:
```

Comment:
Instead of checking the presence of key `apis`, i would check if it is an empty list. As `apis` is a required field, empty `apis` still makes a valid spec and kinda shows that it is empty for a purpose.